app/services/finance/clawback_declaration.rb

```diff
@@ -8,14 +8,16 @@ class ClawbackDeclaration
     validate :validate_refundable_state
     validate :output_fee_statement_available
 
-    def initialize(participant_declaration)
+    def initialize(participant_declaration, voided_by_user:)
```

Comment:
can voiding by `nil` be the default so we don't need to change the interface everywhere?

Reply:
So I did this intentionally; if someone goes to call this service by the rails console I wanted it to throw out an error if they forgot to specify a `voided_by_user`, rather than defaulting to the API-type behaviour. It's less likely to to be misused this way IMO

Reply:
fair enough